### PR TITLE
feat(MCPE-3601): Add scannerInactiveDaysBeforeRemoval property

### PR DESCRIPTION
**Pull request by vicsvet**

### CODE REVIEW

**Review by VisVladimir — APPROVED** [review]

File: customer-config/scan-and-go/sng.device-api-settings.v1.json [hunk 1]
```diff
@@ -10,6 +10,12 @@
       "minimum": 0,
       "maximum": 100,
       "description": "The minimum battery level required to release a handheld scanner."
+    },
+    "scannerInactiveDaysBeforeRemoval": {
```
Comment (khamooshi) [inline customer-config/scan-and-go/sng.device-api-settings.v1.json:14]: please rename to `hhsInactiveDaysBeforeRemoval` to have consistence naming considering the other setting name which is `hhsMinimumBatteryLevelRequiredToRelease`

File: customer-config/scan-and-go/sng.device-api-settings.v1.json [hunk 2]
```diff
@@ -10,6 +10,12 @@
...
+      "type": "integer",
+      "minimum": 0,
+      "maximum": 180,
+      "description": "The amount of days after which inactive handheld scanner is removed. Default value 0: scanners won't be removed."
```
Comment (khamooshi) [inline customer-config/scan-and-go/sng.device-api-settings.v1.json:18]: Number of days after which an inactive handheld scanner is removed. The default is set to 90 days.

Comment (khamooshi) [inline customer-config/scan-and-go/sng.device-api-settings.v1.json:18]: Default value should not be zero, it should be 90 days.

**Review by khamooshi — CHANGES_REQUESTED** [review]

File: customer-config/scan-and-go/sng.device-api-settings.v1.json [hunk 3]
```diff
@@ -10,6 +10,13 @@
       "minimum": 0,
       "maximum": 100,
       "description": "The minimum battery level required to release a handheld scanner."
+    },
+    "hhsInactiveDaysBeforeRemoval": {
+      "type": "integer",
+      "default": 90,
+      "minimum": 0,
+      "maximum": 180,
+      "description": "The amount of days after which inactive handheld scanner is removed. Default value: 90"
```
Comment (khamooshi) [inline customer-config/scan-and-go/sng.device-api-settings.v1.json:19]: Would you change the description to 
"Number of days after which an inactive handheld scanner is removed. The default is set to 90 days."

and please merge after this change if someone approves

**Review by khamooshi — APPROVED** [review]

**Pull request merged**